.github/workflows/release.yml

```
@@ -0,0 +1,292 @@
+        run: |
+          set -euo pipefail
+          paths=$(find . -maxdepth 1 -type f -name 'cordum-*' | sort | sed 's|^./||')
+          if [ -z "$paths" ]; then
+            echo "::error::no cordum-* artefacts staged for manifest"
+            exit 1
+          fi
+          ./.manifest-cli $paths > SHA256SUMS
+          rm -f .manifest-cli
+          cat SHA256SUMS
+      - name: Detach-sign SHA256SUMS (Tier 1)
+        if: ${{ secrets.GPG_RELEASE_KEY_PRIVATE != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)